upcase_booklist_app/templates/author.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html>
+
+    <head>
+        <title>Curious Programmer | Author</title>
+        <link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+
+    <body> 
+        <nav class="page-header">
+            <div class="header-items"><a href="{{ url_for('index') }}">Home</a></div> 
+            <div class="header-items"><a href="{{ url_for('book_list') }}">Books</a></div>
+            <div class="header-items"><a href="{{ url_for('authors') }}">Authors</a></div>
+            <div class="header-items"><a href="{{ url_for('lists') }}">Lists</a></div>
+        </nav>
+    <body> 
+
+        <!-- Outer div to contain everything -->
+        <div>
+
+            <div class="container">
+                <!-- One parent -->
+                <div class="author-name">
+                    <h1 class="name">{{ author_object['name'] }}</h1>
+                    <p class="text">{{ author_object['bio'] }}</p>
+                </div>
+
+
+                <div class="author-details">
+                    {% for bio_details in author_object['books'] %}
+                    
+                    <div class="book-detail-spacing">       
+                        <a href="{{ url_for('book', book_id=author_object['author_id']) }}">
```

Comment:
Should the value passed to `book_id` be the book's id? It appears you're passing the author's id.